apps/api/v2/src/ee/me/me.controller.ts

```diff
@@ -10,23 +16,35 @@ import { ApiResponse } from "@calcom/platform-types";
   path: "me",
   version: "2",
 })
+@UseGuards(AccessTokenGuard)
 export class MeController {
+  constructor(private readonly usersRepository: UsersRepository) {}
+
   @Get("/")
-  @UseGuards(AccessTokenGuard)
-  async getMe(@GetUser() user: User): Promise<ApiResponse<UserReturned>> {
-    const TWELVE_HOUR_TIME_FORMAT = 12;
+  async getMe(@GetUser() user: User): Promise<ApiResponse<{ user: UserResponse }>> {
+    const me = userSchemaResponse.parse(user);
 
     return {
       status: SUCCESS_STATUS,
       data: {
-        id: user.id,
-        email: user.email,
-        timeFormat: user.timeFormat || TWELVE_HOUR_TIME_FORMAT,
-        defaultScheduleId: user.defaultScheduleId,
-        weekStart: user.weekStart,
+        user: me,
       },
     };
   }
-}
 
-export type UserReturned = Pick<User, "id" | "email" | "timeFormat" | "defaultScheduleId" | "weekStart">;
+  @Put("/")
```

Comment:
Good catch - refactored me, schedules, oauth clients and oauth client users endpoints to use patch instead of put in [refactor: patch instead of put](https://github.com/calcom/cal.com/pull/13486/commits/ca582119974a21e27ed647396ed7b0b3e5e11144)